alerts_back_out.sh

```diff
@@ -0,0 +1,81 @@
+#! /bin/sh
+# Run on r2d2-v because dump monitor and acorn scripts are on /data/mta
+# Run as user mta
+
+git_dir=/home/malgosia/git/alerts_update
+
+# Space weather
+swdir=/data/mta4/space_weather
```

Comment:
I assume these have been tested on the side by setting these destination directories to testing dirs.